common/random/lcg.ksy

```diff
@@ -0,0 +1,42 @@
+meta:
+  id: lcg
+  title: Linear Congruential Generator
+  xref:
+    iso: "28640:2010"
+    wikidata: Q1190228
+  license: Unlicense
+  endian: le
+doc: |
+  A simple non-secure pseudorandom generator.
```

Comment:
A very interesting idea (and certainly very clever hack with this recursive approach to keep state). What we'll definitely need is:

* A better name. "LCG" is "London Capital Group", "Living Card Game", "Large Hadron Collider Computing Grid", "London Computer Group", certain airports, train stations, venues, and probably tons of other things. Even `random_lcg` will be a better name that won't clash with all that.
* More documentation — at least a very basic doc that explains what's going on and how to use it both from KSY author perspective and from end user's perspective.